src/test/scala/uk/gov/nationalarchives/fileformat/FFIDExtractorTest.scala

```diff
@@ -47,11 +47,22 @@ class FFIDExtractorTest extends AnyFlatSpec with MockitoSugar with EitherValues
     m.puid.isEmpty should be(true)
   }
 
+  "the ffid method" should "return a file extension mismatch if one exists" in {
```

Comment:
Could you add a test for the "format name" results as well: `"the ffid method" should "return the format name if one exists"`